sources/dylan/integer.dylan

```diff
@@ -272,16 +272,16 @@ define integer-binary-logical binary-logior machine-word-logior identity;
 define integer-binary-logical binary-logand machine-word-logand identity;
 define integer-binary-logical binary-logxor machine-word-logxor force-integer-tag;
 
-define inline function lognot (x :: <integer>) => (result :: <integer>)
+define sealed inline method lognot (x :: <integer>) => (result :: <integer>)
```

Comment:
Since this is a method now and not a function, should we explicitly specify a generic for it?

(This also goes for `logbit?`, `binary-logior`, `binary-logand`, `binary-logxor`, `ash`, `ash-right`, and `lsh`.)